frontend/src/components/table/TableCell.js

```diff
@@ -224,15 +104,95 @@ class TableCell extends PureComponent {
     } = this.props;
     const widgetData = getWidgetData(item, isEditable, supportFieldEdit);
 
-    if (isEditable) {
-      handleDoubleClick(e, property, true, widgetData[0]);
-    }
+    isEditable && handleDoubleClick(e, property, true, widgetData[0]);
   };
 
+  /**
+   * @method clearValue
+   * @summary Set `clearWidgetValue` value based on a given `reset` param
+   * {string|null} reset
+   */
   clearValue = (reset) => {
-    this.clearWidgetValue = reset == null ? true : false;
+    this.clearWidgetValue = reset === null ? true : false;
```

Comment:
I think this was `== null` because it's a falsy value check, and not a strict comparison. So worth checking what `clearValue` can be called with to be sure it's ok to make this change.

Reply:
Reverting as you are right. When is undefined  it this.clearWidgetValue will be true. I've checked the calls in the TableItem

Reply:
@petrican  why not documenting this inside `clearValue` function because it seems it wasn't obvious.
Also I think it would make sense to document the `reset` parameter.

Reply:
Done.